developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -96,6 +98,8 @@ The following properties are supported:
 - :code:`{http://nextcloud.org/ns}has-preview`
 - :code:`{http://owncloud.org/ns}size` Unlike :code:`getcontentlength`, this property also works for folders reporting the size of everything in the folder.
 - :code:`{http://nextcloud.org/ns}rich-workspace` this property is provided by the text app
+- :code:`{http://nextcloud.org/ns}contained-folder-count` The number of folders contained in the folder
```

Comment:
Maybe additionally clarify that the counting is only on the first depth level, not recursively counting files/folders.

Reply:
:+1: